go.mod

```diff
@@ -81,7 +81,10 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/autoscaler/vertical-pod-autoscaler v0.0.0-20191115143342-4cf961056038 // indirect
+	k8s.io/klog v1.0.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20230308215209-15aac26d736a // indirect
+	k8s.io/kube-state-metrics v1.9.8 // indirect
```

Comment:
Why are you adding v1 as a dependency here?

Reply:
@mrueg Thanks for your review! I think this got added when I did a `go mod tidy` and I didn't notice, my mistake. Let me remove this and push again.

Reply:
I just need to remove that one line right? (`k8s.io/kube-state-metrics v1.9.8 // indirect`)

Reply:
Actually, no, it got added when I ran `go get k8s.io/kube-state-metrics`.